paper-scroll-header-panel.html

```diff
@@ -295,7 +295,7 @@
       'iron-resize': 'measureHeaderHeight'
     },
 
-    ready: function() {
+    attached: function() {
       this.async(this.measureHeaderHeight, 5);
```

Comment:
I will only move `this.async(this.measureHeaderHeight, 1);` to `attached` and keep the rest of the code in `ready` since this element doesn't remove the event listener on detached.